source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 
-The auditor must force 50 Limit_Exceeded turnaways.
+The Title Report will be COUNTER-compliant if the following audit tests are passed. The book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3) will be COUNTER-compliant if the metrics match those in the Title Report, with the appropriate aggregation.
 
-* The auditor will log into the site. They will select and make a search on a database or browse to a database causing the user limit to reach the maximum allowance. The auditor will then log into the same site using a different computer. The auditor will then repeat the action made on the previous computer (select and make a search on a database or browse to a database).
-* The user should then be refused access as the concurrent/simultaneous-user limit has exceeded. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Audit test requirements vary depending on the set up of the platform and any related database(s), as indicated by the Options within the tests below.
 
-Each of these concurrent/simultaneous access denials must report 1 Limit_Exceeded in the DR_D2 Standard View.
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
 
-**Option 3**: The content provider denies the user access when the concurrent/simultaneous-user limit is exceeded upon accessing an item within a database.
+E.2.4.1 Unique_Title_Requests
+'''''''''''''''''''''''''''''
 
-The auditor must force 50 Limit_Exceeded turnaways.
+**Option 1**: Book_Segments and/or Reference_Items are available, users can elect to access Books and/or Reference_Works segment-by-segment.
 
-* The auditor will log into the site and will navigate to and request an item. This will cause the user limit to reach the maximum allowance.The auditor will log into the site again using a different computer. The auditor will then repeat the action made on the previous computer (navigate to and request an item).
-* After the item has been requested the user should then be denied access. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+The auditor MUST request 100 Book_Segments and/or Reference_Items, 10 each from 10 different Books and/or Reference_Works. This MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 10 each of Unique_Title_Investigations and Unique_Title_Requests.
 
-Each of these concurrent/simultaneous turnaways must report 1 Limit_Exceeded in the DR_D2 Standard View.
+**Option 2**: Only whole Books and/or Reference_Works are available.
 
+The auditor MUST request 20 Books and/or Reference_Works, twice each. This MUST result in 40 each of Total_Item_Investigations and Total_Item_Requests, and 20 each of Unique_Item_Investigations, Unique_Item_Requests, Unique_Title_Investigations and Unique_Title_Requests.
```

Comment:
This is only the case when the Book_Segments/Reference_Items cannot be identified, otherwise the number of Total/Unique_Item_Investigations and Requests depend on the number of Book_Segments/Reference_Items in the downloaded titles.